src/test/java/tdl/datapoint/coverage/CoverageDatapointAcceptanceTest.java

```diff
@@ -116,6 +117,7 @@ public void create_repo_and_uploads_commits() throws Exception {
         assertThat(languageEvent.getParticipant(), equalTo(participantId));
         assertThat(languageEvent.getChallengeId(), equalTo(challengeId));
         assertThat(languageEvent.getProgrammingLanguage(), equalTo("HmmmLang"));
+//        assertThat(languageEvent.getProgrammingLanguage(), equalTo("Java"));
```

Comment:
Thanks for spotting the problem with the Hmmm container test.
These comments should be removed as they are not needed.